pkg/flare/processflare/archive_process.go

```diff
@@ -0,0 +1,20 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build process
+
+// Package processflare contains flare logic that only imports pkg/process when the process build tag is included
+package processflare
```

Comment:
Can we add a new package `pkg/flare/sysprobe`? We should also get rid of the `process` build tag.

Reply:
Okay, will update - also, is there a more suitable build tag I can use? I'm using `process` to avoid this getting built into other agents like the security agent (to resolve [this comment](https://github.com/DataDog/datadog-agent/pull/29768#issuecomment-2396409445)

Reply:
Is there one for the core agent? Otherwise `process` is fine.

Reply:
Not that I've seen, it would be easy to get rid of `process` entirely if that existed though, since the only other consumer of the `process` tag is the sysprobe status flare logic associated with `getSystemProbeStats`, that existed before